plugin/lsp.vim

```diff
@@ -63,7 +64,8 @@ command! LspPreviousReference call lsp#ui#vim#references#jump(-1)
 
 nnoremap <plug>(lsp-code-action) :<c-u>call lsp#ui#vim#code_action()<cr>
 nnoremap <plug>(lsp-declaration) :<c-u>call lsp#ui#vim#declaration()<cr>
-nnoremap <plug>(lsp-definition) :<c-u>call lsp#ui#vim#definition()<cr>
+nnoremap <plug>(lsp-definition) :<c-u>call lsp#ui#vim#definition(0)<cr>
+nnoremap <plug>(lsp-peek-definition) :<c-u>call lsp#ui#vim#definition(1)<cr>
```

Comment:
Could we use `v:true` and `v:false` to express more clearly that it is boolean? Or does it really require an int somewhere I missed? (same goes for the command)

Reply:
Old version of neovim doesn’t support v:true and it is buggy too.

Reply:
okay, didn't know that